escrow/transactions/admin/leaderboards/withdraw_entry.cdc

```diff
@@ -1,15 +1,19 @@
 import Escrow from "../../../contracts/AllDay.cdc"
 import AllDay from "../../../contracts/AllDay.cdc"
+import NonFungibleToken from "../../../contracts/NonFungibleToken.cdc"
 
 // This transaction takes the leaderboardName and nftID and returns it to the correct owner.
-transaction(leaderboardName: String, nftID: UInt64) {
+transaction(leaderboardName: String, nftID: UInt64, ownerAddress: Address) {
     prepare(signer: AuthAccount) {
         // Get a reference to the Collection resource in storage.
         let collectionRef = signer.borrow<&Escrow.Collection>(from: Escrow.CollectionStoragePath)
             ?? panic("Could not borrow reference to the Collection resource")
 
+        let depositCap = getAccount(ownerAddress)
+            .getCapability<&{NonFungibleToken.CollectionPublic}>(AllDay.CollectionPublicPath)
+
```

Comment:
@jrkhan do you mean `AllDay.CollectionPublicPath` should be gotten from the leaderboard?

Reply:
synced offline with Jamil. Let's add an argument to [addEntryToLeaderBoard](https://github.com/dapperlabs/studio-platform-smart-contracts/pull/45/files#diff-f5756b642ceb5551b34b8028cfc5e7cdf3a2df886c661a7568f467ea0241edc8R52) that takes in a PublicPath argument that will be used to deposit the nft and store it as part of the Escrow entry. This way the deposit cap can be created in [this function](https://github.com/dapperlabs/studio-platform-smart-contracts/pull/45/files#diff-f5756b642ceb5551b34b8028cfc5e7cdf3a2df886c661a7568f467ea0241edc8R77) instead of passed to it